.npmignore

```diff
@@ -1 +0,0 @@
-src/
```

Comment:
Ah, the tests were ending up in `dist/tests` as well, which is why I went with the whitelist approach.